pkg/function/kubeops_test.go

```diff
@@ -241,6 +241,41 @@ func (s *KubeOpsSuite) TestKubeOps(c *C) {
 	}
 }
 
+func (s *KubeOpsSuite) TestKubeOpsCreateDeleteWithCoreResource(c *C) {
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+	defer cancel()
+	tp := param.TemplateParams{}
+	action := "test"
+	gvr := schema.GroupVersionResource{Group: "", Version: "v1", Resource: "services"}
+	serviceName := "test-deployment-2"
+
+	bp := newCreateResourceBlueprint(deletePhase(gvr, serviceName, s.namespace))
```

Comment:
It seems to me that this test is attempting deletion of a resource created in the previous test. It means that one must always run the previous test before running this one which is not a good test design. When run by itself, this test will always fail.
Please modify this to create and delete the resource so that it can be run independently